ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/EditOwner.cshtml

```diff
@@ -0,0 +1,95 @@
+﻿@page "/case/{urn:long}/management/edit-case-owner"
```

Comment:
The styling of this page isn't quite correct, can you work with Israt to touch it up

The display value of the options needs to have the name and the email